openfisca_us/parameters/gov/states/in/tax/income/taxes/agi/rates.yaml

```
@@ -0,0 +1,11 @@
+description: IN adjusted gross income tax rate.
+values:
+  2021-01-01: 0.0323
```

May as well since it's in the legal code
```suggestion
  2014-01-01: 0.034 # "For taxable years beginning before January 1, 2015"
  2015-01-01: 0.033
  2017-01-01: 0.0323
```